rest/src/main/java/org/jboss/pnc/rest/provider/ArtifactProvider.java

```diff
@@ -181,6 +185,13 @@ private String getPublicUrl(Artifact artifact) {
         return queryForCollection(pageIndex, pageSize, sortingRsql, query, withDependantBuildRecordId(buildRecordId));
     }
 
+    public CollectionInfo<ArtifactRest> getAll(int pageIndex, int pageSize, String sortingRsql, String query,
+            Optional<String> sha256, Optional<String> md5, Optional<String> sha1) {
+        logger.debug("HELLO + sha1: " + sha1 + " sha256:" + sha256 + " md5: " + md5);
```

Comment:
Better debug message would be nice ;-)

Reply:
oh, I was supposed to delete that :D you've seen nothing!